### PR TITLE
Javascript-javascript1-week2/eva

### PR DESCRIPTION
**Pull request by yagmureva**
done

### CODE REVIEW

File: 3_javascript/javascript1/week2/script.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
+
+//Flight booking fullname function
+
+// Define the getFullname function
+function getFullname(firstname, surname) {
+  return firstname + " " + surname;
```
Comment (elilovna) [inline 3_javascript/javascript1/week2/script.js:6]: You could try to use template literals here  😉
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: 3_javascript/javascript1/week2/script.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+}
+
+// Call the getFullname function and assign the results to variables
+var fullname1 = getFullname("Benjamin", "Hughes");
```
Comment (elilovna) [inline 3_javascript/javascript1/week2/script.js:10]: it is better to use const or let here

File: 3_javascript/javascript1/week2/script.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+var fullname2 = getFullname("John", "Smith");
+
+// Log out the two fullname variables
+console.log(fullname1); // "Benjamin Hughes"
+console.log(fullname2); // "John Smith"
+function getFullname(firstname, surname, useFormalName = false, isFemale = false) {
+  if (useFormalName && firstname !== "") {
+    if (isFemale) {
+      return "Lady " + firstname + " " + surname;
```
Comment (elilovna) [inline 3_javascript/javascript1/week2/script.js:19]: you could try to use template literals

File: 3_javascript/javascript1/week2/script.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+//Candy helper optional
+
+// Initialize an array to store candy prices
+const boughtCandyPrices = [];
+
+// Function to add candy and its price to the array
+function addCandy(candyType, weight) {
+  let pricePerGram = 0;
+
+  switch (candyType) {
```
Comment (elilovna) [inline 3_javascript/javascript1/week2/script.js:149]: nice that you used switch

**Review by elilovna — CHANGES_REQUESTED** [review]
I see that you learned material, Very good work 😎

**Comment by shpomp** [comment]
image

**Pull request closed without merging**